src/main/java/org/threeten/extra/AmountFormats.java

```diff
@@ -287,6 +307,186 @@ private static boolean oppositeSigns(int a, int b) {
         return a < 0 ? (b >= 0) : (b < 0);
     }
 
+    // -------------------------------------------------------------------------
+    /**
+     * Parses a unit-based duration string to a duration value.
+     * <p>
+     * This is an implementation of Golang's {@link https://golang.org/pkg/time/#ParseDuration}:
```

Comment:
Maybe rephrase something more like "The parses formatted durations based on units. The behaviour matches that of the <a href="https://golang.org/pkg/time/#ParseDuration">Golang</a> duration parser."

Reply:
Done.